src/core/server/http/http_config.ts

```diff
@@ -38,7 +39,20 @@ export const config = {
           validate: match(validBasePathRegex, "must start with a slash, don't end with one"),
         })
       ),
-      defaultRoute: schema.maybe(schema.string()),
+      csp: schema.object(
```

Comment:
If I understand correctly, all values set in **kibana.yml** under the `csp` section are ignored. 
Because in the new platform we read from `server.csp` section.